### ISSUE TITLE
Mapper map data not available for subsequent activities

### ISSUE THREAD

**Issue by neethuabacker** [TIBCOSoftware/flogo#513]
**Current behavior (how does the issue manifest):**
Currently mapper only has map to flow output. 

**Expected behavior:**
If we would like to merge or collate response or do some data transformations in mapper and then use that data in a subsequent activity like REST reply its not allowing us
there is a totally different version of mapper available in flogo contrib, it has "map input" but doesnt allow me to define a json to the input and settings option is disabled. 

**Minimal steps to reproduce the problem (not required if feature enhancement):**

**Please tell us about your environment (Operating system, docker version, browser & web ui version, etc):**

**Flogo version (CLI & contrib/lib. If unknown, leave empty or state unknown):** 0.X.X
Flogo v0.9.2

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by mellistibco** [comment]
@neethuabacker yes, we've run into this several times. We're able to workaround this by leveraging subflows, however thats probably not an ideal solution depending on the complexity of your app. We'll look into enhancing the mapper activity.
